cicecore/cicedynB/dynamics/ice_dyn_shared.F90

```diff
@@ -162,6 +163,10 @@ subroutine init_dyn (dt)
          ! velocity
          uvel(i,j,iblk) = c0    ! m/s
          vvel(i,j,iblk) = c0    ! m/s
+         if (grid_system == 'CD') then ! extra velocity variables
```

Comment:
Correct.  I figured that was the easiest thing to do for now.  And I added uveln and vvele as the "extra" arrays for the CD version.  This may not be what we want, but it's a starting point.